doc/man3/RSA_get0_key.pod

```diff
@@ -82,10 +110,31 @@ needed, duplicate the received value using BN_dup() and pass the
 duplicate.  The same applies to RSA_get0_factors() and RSA_set0_factors()
 as well as RSA_get0_crt_params() and RSA_set0_crt_params().
 
+The size for the return array of RSA_get0_multi_prime_factors() and
+RSA_get0_multi_prime_params() is not given by those two functions.
+The caller should obtain the size by calling RSA_get_multi_prime_extra_count()
+in advance and allocate sufficient buffer to store the return values before
+calling RSA_get0_multi_prime_factors() and RSA_get0_multi_prime_params().
+
+RSA_set0_multi_prime_params() always clears the original multi-prime
+triplets in RSA object B<r> and assign the new set of triplets into it.
+
+If an RSA object is newly created and its multi-prime parameters are set
+manually by using functions described in this document, the RSA_set_version()
+function should be called with B<ver> set to B<RSA_ASN1_VERSION_MULTI> before
+any other 'actual' operation on this RSA object (e.g., check, encryption,
+decryption and etc).
+
 =head1 RETURN VALUES
 
-RSA_set0_key(), RSA_set0_factors and RSA_set0_crt_params() return 1 on
-success or 0 on failure.
+RSA_set0_key(), RSA_set0_factors(), RSA_set0_crt_params(), RSA_set_version()
+and RSA_set0_multi_prime_params() return 1 on success or 0 on failure.
+
+RSA_get0_multi_prime_factors() and RSA_get0_multi_prime_crt_params() reutrn
+1 on success or 0 on failure.
+
+RSA_get_multi_prime_extra_count() returns the number of other primes or 0 if
```

Comment:
I'd say that "additional" instead of "other" would be better here. But there might be better word, "excess" perhaps? Ping to English speakers...

Reply:
I prefer 'additional' since the RFC uses this word...

Reply:
s/reutrn/return/

Reply:
One could also use a construction like "two less than the number of primes in use, which is 0 for traditional RSA and the number of extra primes for multi-prime RSA".

Reply:
I'd say it's appropriate to throw in few words along Ben's suggestion. Thing is that we probably can't expect every reader to be intimately familiar with RFC text to recognize 'additional' as specific term.

Reply:
Thrown in.